src/data_coordinator/unit_test/CMakeLists.txt

```
@@ -0,0 +1,13 @@
+set_full_path(THIS_DIR_SEQ_CATCH2_TEST_FILES
+  )
+
+set_full_path(THIS_DIR_MPI_CATCH2_TEST_FILES
+  data_coordinator_HDF5_hrrl_public_api.cpp
+  )
+
+set(LBANN_SEQ_CATCH2_TEST_FILES
+  "${LBANN_SEQ_CATCH2_TEST_FILES}"
+  "${THIS_DIR_SEQ_CATCH2_TEST_FILES}" PARENT_SCOPE)
+set(LBANN_MPI_CATCH2_TEST_FILES
+  "${LBANN_MPI_CATCH2_TEST_FILES}"
+  "${THIS_DIR_MPI_CATCH2_TEST_FILES}" PARENT_SCOPE)
```

```suggestion
set_full_path(THIS_DIR_MPI_CATCH2_TEST_FILES
  data_coordinator_HDF5_hrrl_public_api.cpp
  )

set(LBANN_MPI_CATCH2_TEST_FILES
  "${LBANN_MPI_CATCH2_TEST_FILES}"
  "${THIS_DIR_MPI_CATCH2_TEST_FILES}" PARENT_SCOPE)
```
Also add the license.